packages/opentelemetry-metrics/src/export/types.ts

```diff
@@ -39,6 +39,13 @@ export enum AggregatorKind {
   HISTOGRAM,
 }
 
+/** The kind of aggregator. */
+export enum AggregationTemporality {
```

Comment:
could we re-use the enum inside the `api-metrics` package instead of duplicating it ?

Reply:
thanks! fixed